torch/onnx/symbolic_opset9.py

```diff
@@ -5848,7 +5848,7 @@ def try_mask_to_index(index):
                     "aten::index",
                     "operator of advanced indexing on tensor of unknown rank. "
                     "Try turning on shape inference during export: "
-                    "torch.onnx._export(..., onnx_shape_inference=True).",
+                    "torch.onnx.utils._export(..., onnx_shape_inference=True).",
```

Comment:
Could you remove

```
                    "Try turning on shape inference during export: "
                    "torch.onnx.utils._export(..., onnx_shape_inference=True)."
```

instead, as we do not want users to use internal apis. shape_inference is enabled by default anyways.